ovh/resource_dedicated_server_reinstall_task.go

```diff
@@ -0,0 +1,467 @@
+																		"name": {
+																			Type:        schema.TypeString,
+																			Optional:    true,
+																			ForceNew:    true,
+																			Description: "zpool name (generated automatically if not specified)",
+																		},
+																	},
+																},
+															},
+														},
+													},
+												},
+												"file_system": {
+													Type:        schema.TypeString,
+													Required:    true,
+													ForceNew:    true,
+													Description: "File system type",
+												},
+												"mount_point": {
+													Type:        schema.TypeString,
+													Required:    true,
+													ForceNew:    true,
+													Description: "Mount point",
+												},
+												"raid_level": {
+													Type:        schema.TypeInt,
+													Optional:    true,
+													ForceNew:    true,
+													Description: "Software raid type (default is 1)",
+												},
+												"size": {
+													Type:        schema.TypeInt,
+													Optional:    true,
+													ForceNew:    true,
+													Description: "Partition size in MiB (default value is 0)",
+												},
+											},
+										},
+									},
+									"disks": {
+										Type:        schema.TypeInt,
+										Optional:    true,
+										ForceNew:    true,
+										Description: "Total number of disks in the disk group involved in the partitioning configuration (all disks of the disk group by default)",
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+			//Computed
+			"comment": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Details of this task",
+			},
+			"done_date": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Completion date",
+			},
+			"function": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Function name",
+			},
+			"last_update": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Last update",
+			},
+			"start_date": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Task Creation date",
+			},
+			"status": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Task status",
+			},
+		},
+	}
+}
+
+func resourceDedicatedServerReinstallTaskImportState(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	//
+	// After creating an reinstall task, there is no way to get the name of the server (service_name)
+	// nor the name of template (operating_system) used during the creation of the task.
+	// This is why it is required to provide them when importing.
+	//
+
+	givenId := d.Id()
+	splitId := strings.SplitN(givenId, "/", 3)
+	if len(splitId) != 3 {
+		return nil, fmt.Errorf("Import Id is not service_name/operating_system/task_id formatted")
```

Comment:
not sure it makes sense to give the operating_system there:
- I'm not convinced that importing this resource makes sense ?
- If we really want to be able to import it, maybe we could manually set a placeholder for this field in this func